### PR TITLE
Use Package Json Prettier Plugin

### PR DESCRIPTION
**Pull request by samchungy**
Resolves https://github.com/seek-oss/skuba/issues/1275

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -160,6 +160,7 @@
   "publishConfig": {
     "provenance": true
   },
+  "readme": "ERROR: No README data found!",
```
Comment (samchungy) [inline package.json:163]: I'm not very sure why these pop up 🤔

File: src/cli/configure/analysis/project.test.ts [hunk 2]
```diff
@@ -5,8 +5,6 @@ import { defaultOpts } from '../testing/module';
 import * as project from './project';
 import { diffFiles } from './project';
 
-jest.mock('sort-package-json');
```
Comment (samchungy) [inline src/cli/configure/analysis/project.test.ts:8]: am happy again

**Comment by samchungy** [comment]
Looks like autofix is brokey

File: src/cli/configure/addEmptyExports.test.ts [hunk 3]
```diff
@@ -27,17 +27,17 @@ describe('tryAddEmptyExports', () => {
     await expect(tryAddEmptyExports()).resolves.toBeUndefined();
 
     expect(writeFile.mock.calls.flat().join('\n')).toMatchInlineSnapshot(`
-      "~/project/jest.setup.ts
-      // jest.setup.ts
+"~/project/jest.setup.ts
```
Comment (samchungy) [inline src/cli/configure/addEmptyExports.test.ts:30]: I broke something 🤔

**Comment by samchungy** [comment]
🌶️ Spicy changeset name tho

File: src/api/git/commitAllChanges.ts
Comment (72636c) [inline src/api/git/commitAllChanges.ts:1]: Is this meant for #1277?

**Review by 72636c — APPROVED** [review]
😌

File: src/api/git/commitAllChanges.ts
Comment (samchungy) [reply src/api/git/commitAllChanges.ts:1]: ✅ yus, was just checking it worked

image

**Pull request merged**